src/effects/Tracker.h

```diff
@@ -0,0 +1,133 @@
+/**
+ * @file
+ * @brief Header file for Tracker effect class
+ * @author Jonathan Thomas <jonathan@openshot.org>
+ *
+ * @ref License
+ */
+
+/* LICENSE
+ *
+ * Copyright (c) 2008-2019 OpenShot Studios, LLC
+ * <http://www.openshotstudios.com/>. This file is part of
+ * OpenShot Library (libopenshot), an open-source project dedicated to
+ * delivering high quality video editing and animation solutions to the
+ * world. For more information visit <http://www.openshot.org/>.
+ *
+ * OpenShot Library (libopenshot) is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * OpenShot Library (libopenshot) is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with OpenShot Library. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef OPENSHOT_TRACKER_EFFECT_H
+#define OPENSHOT_TRACKER_EFFECT_H
+
+#include "../EffectBase.h"
+
+#include <google/protobuf/util/time_util.h>
```

Comment:
Could this be moved to the `Tracker.cpp` file (along with the `using google::protobuf::util::TimeUtil;` farther down)? Same for the other headers that include it.

I don't see any actual use of those symbols in the headers (unless I'm missing it), and SWIG is complaining because it doesn't know what `google::protobuf::util::TimeUtil` is. (Since it doesn't get those headers.)